server/app.js

```diff
@@ -36,4 +37,16 @@ app.use(function(err, req, res, next) {
   res.json({ error: err });
 });
 
+mongoose
+  .connect(
+    "mongodb://localhost:27017/tlc?readPreference=primary&appname=MongoDB%20Compass&ssl=false",
```

Comment:
This should be an environment variable in .env file
We can also gitignore the .env file now and add an example .env file for new developers to follow

Reply:
yeah thats how i did it for my project. because our .env file is visible too so i left it here . once everyone know how the connection string looks like, i think we can gitignore .env file because we probably need more secret keys to put there, or i can create config file and move it there.